impl/src/planning.rs

```diff
@@ -74,6 +76,8 @@ struct DependencySet {
   build_deps: Vec<BuildableDependency>,
   // Dependencies that are required for tests
   dev_deps: Vec<BuildableDependency>,
+  // Dependencies that have been renamed and need to be aliased in the build rule
+  renamed_deps: HashMap<String, String>,
```

Comment:
It doesn't look like you use the lookup into this hashmap.  Could this, instead, be a Vec of a struct, similar to [DependencyAlias](https://github.com/google/cargo-raze/pull/123/commits/fde76fdffe5fc645a41c4615486191b3330da27a#diff-b839b62c6331bbd8524ee98a856d496eR57-R61) here?

Reply:
I went ahead and changed it to a Vec<String>. Because of this, I moved the logic that creates the alias strings for the template into `produce_deps`